odkx-src/sync-endpoint-manual-setup.rst

```diff
@@ -52,62 +52,66 @@ Setup instructions:
 
   .. code-block:: console
 
-    $ git clone https://github.com/odk-x/sync-endpoint
-   
-  3. Navigate into the sync-endpoint directory. Most likely
+    $ git clone -b master --single-branch --depth=1 https://github.com/odk-x/sync-endpoint
+
+  .. note::
+
+     The above command performs a shallow clone of the sync-endpoint Github repository. To perform a full clone remove :guilabel:`--single-branch --depth=1` from the command.
```

Comment:
It is a git repository, not a GitHub repository. Or you could specify that it's a GitHub git repository.